test_util/cluster_api.py

```diff
@@ -525,7 +288,7 @@ def metronome_one_off(self, job_definition, timeout=300, ignore_failures=False):
                         retry_on_result=lambda ret: not ret,
                         retry_on_exception=lambda x: False)
         def wait_for_completion():
-            r = self.get('/service/metronome/v1/jobs/' + job_id, params={'embed': 'history'})
+            r = self.metronome.get('/jobs/' + job_id, params={'embed': 'history'})
```

Comment:
no leading slash needed here (and the other metronome ones) I think because get_client() includes a `/` at the end.